package.json

```diff
@@ -27,11 +27,12 @@
     "eslint-plugin-react-perf": "^3.3.1",
     "eslint-plugin-tsdoc": "^0.2.17",
     "eslint-plugin-unused-imports": "^2.0.0",
+    "json": "^11.0.0",
```

Comment:
This is the[ lib that I use](https://github.com/trentm/json) to edit the package.json using cli.